### PR TITLE
Check LMS connection before sending uptime request

### PR DESCRIPTION
**Pull request by alex-deaconu**
Remove lenghthy log messages

### CODE REVIEW

**Comment by alex-deaconu** [comment]
From our testing, when the network is down the Display can no longer send messages through LMS, so Content Uptime requests fail. This is happening despite the content playback seemingly being normal.

Because this of this we could get inaccurate failure reports, we discussed and decided to disable Content Uptime when LMS is down.

@ezequielc @andrecardoso please review. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu 👍

File: src/main/uptime/content-uptime.js [hunk 1]
```diff
@@ -55,13 +55,23 @@ function init() {
...
 function retrieveUptime() {
   if (playingItem && playingItem.presentationType === "HTML Template") {
-    commonMessaging.broadcastToLocalWS({
-      from: "player-electron",
-      topic: "content-uptime", 
-      forPresentationId: playingItem.presentationId
+    commonMessaging.checkMessagingServiceConnection()
+    .then(msResult=>{
+      if (msResult !== 'connected') {
+        throw(msResult);
```
[the remark this answers is not in the document]
Comment (alex-deaconu) [reply src/main/uptime/content-uptime.js]: @andrecardoso good spot, great suggestion! Updated, thanks.

**Comment by andrecardoso** [comment]
[(0) Electron -As Player, I provide an uptime logging mechanism for components](https://trello.com/c/AzrraQQU/5374-0-electron-as-player-i-provide-an-uptime-logging-mechanism-for-components)

**Pull request merged**